client/components/tinymce/plugins/contact-form/dialog/field-edit-button.jsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes, PureComponent } from 'react';
+import classNames from 'classnames';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Gridicon from 'components/gridicon';
+import Popover from 'components/popover';
+
+class ContactFormDialogFieldEditButton extends PureComponent {
+	static propTypes = {
+		expanded: PropTypes.bool.isRequired
+	};
+
+	constructor() {
+		super();
+		this.state = { showTooltip: false };
+	}
+
+	render() {
+		const { translate } = this.props;
```

Comment:
Optional but, since we're destructuring we can also do:
```javascript
const { expanded, translate } = this.props;
//...
'is-expanded': expanded
```